modules/sdk/project-templates/project-templates-npm-billboardjs-portlet/src/main/resources/archetype-resources/src/main/java/portlet/__className__Portlet.java

```diff
@@ -26,4 +35,30 @@
 	service = Portlet.class
 )
 public class ${className}Portlet extends MVCPortlet {
+
+	@Override
+	public void doView(
+		RenderRequest renderRequest, RenderResponse renderResponse)
+			throws IOException, PortletException {
+
+		JSPackage jsPackage = _npmResolver.getJSPackage();
+
+		renderRequest.setAttribute(
+			"bootstrapRequire",
+			jsPackage.getResolvedId() + " as bootstrapRequire");
+
+		String contextPath = renderRequest.getContextPath();
+
+		renderRequest.setAttribute(
+			"stylesheetURL",
+			contextPath + "/node_modules/" +
+				_npmResolver.resolveModuleName("billboard.js") +
+				"/dist/billboard.css");
```

Comment:
Shouldn't this line have an extra tab?

Reply:
SF doesn't test it: https://github.com/liferay/liferay-portal/blob/master/modules/sdk/source-formatter.properties#L24